vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
If we kick out rails-assets-react from the load path, we use the react_rails one and just have to specify `config.react.addons = true` wherever we want to use it, no?